client/state/posts/reducer.js

```diff
@@ -141,21 +141,15 @@ export function queries( state = {}, action ) {
 	switch ( action.type ) {
 		case POSTS_REQUEST_SUCCESS: {
 			const { siteId, query, posts, found } = action;
-			if ( ! state[ siteId ] ) {
-				state[ siteId ] = new PostQueryManager();
```

Comment:
>Accidental mutations are really easy to miss.

Indeed! Though in this case, better use of `deepFreeze` in tests would have caught the issue.